dca/tl_module.php

```diff
@@ -14,8 +14,10 @@
  * Palettes
  */
 $GLOBALS['TL_DCA']['tl_module']['palettes']['__selector__'][]       = 'customLanguage';
-$GLOBALS['TL_DCA']['tl_module']['palettes']['changelanguage']       = '{title_legend},name,headline,type;{config_legend},hideActiveLanguage,hideNoFallback,customLanguage;{template_legend:hide},navigationTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID,space';
+$GLOBALS['TL_DCA']['tl_module']['palettes']['__selector__'][]       = 'limitToRootPages';
```

Comment:
I think `limitWebsiteRoots` would be a better naming (since Website root is used as name by contao itself)

Reply:
see b31236f